simple-page-ordering.php

```diff
@@ -186,7 +186,7 @@ public static function ajax_simple_page_ordering() {
 		 * @param int   $start    The start index.
 		 * @param array $excluded Array of post IDs.
 		 *
-		 * @return obj|WP_Error
+		 * @return object|WP_Error|"children"
```

Comment:
@szepeviktor I think we should follow phpDoc here to list `string` instead of `"children"`.

Reply:
@dinhtungdu It is up to you!
There is an RFC for that https://wiki.php.net/rfc/is_literal
and PHPStan implemented literal strings.

Reply:
Thanks for the link! Merging as it is.